response_writer_test.go

```diff
@@ -156,11 +156,8 @@ func TestResponseWriterHijack(t *testing.T) {
 func TestResponseWriteHijackNotOK(t *testing.T) {
 	hijackable := new(http.ResponseWriter)
 	rw := NewResponseWriter(*hijackable)
-	hijacker, ok := rw.(http.Hijacker)
```

Comment:
This test seems to be inverted now. Doesn't that indicate that we no longer implement hijackable when the wrapped writer does?

Reply:
Ah, never mind, I misread this. I see this is testing that it returned an error, previously, if the wrapped response writer didn't implement `Hijacker`.